.github/workflows/learn-github-actions.yml

```diff
@@ -0,0 +1,13 @@
+name: learn-github-actions
+run-name: ${{ github.actor }} is learning GitHub Actions
+on: [push]
+jobs:
+  check-bats-version:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Groups together all the steps that run in the `check-bats-version` job
- action or shell script